docs/UserGuide.md

```diff
@@ -356,58 +349,104 @@ A lesson **must** have the following fields: a start date, a time range, a lesso
+            <td><code>subject/Social studies</code></td>
+        </tr>
+        <tr>
+            <td>Time range</td>
+            <td><code>time/</code></td>
+            <td><ul>
+            <li>Formatted as <code>HHmm-HHmm</code></li>
+            <li>Must be between 8am and 10pm, inclusive</li>
+            </ul></td>
+            <td><code>time/1730-1830</code></td>
+        </tr>
+        <tr>
+            <td>Lesson rate</td>
+            <td><code>rates/</code></td>
+            <td><ul>
+              <li>Fee of the lesson per hour. This rate will be used in the calculation of fees due after each lesson.</li>
+              <li>Refer to <a href="#monetary-fields">Monetary fields</a> for more details.</li>
+            </ul></td>
+            <td><code>rates/50</code></td>
+        </tr>
+        <tr>
+        <td rowspan=5>Optional</td>
+            <td>Recurrence flag</td>
+            <td><code>recurring/</code></td>
+            <td><ul>
+              <li>Indicates whether the lesson is recurring</li>
+              <li>Optional parameter: end date of the recurrence</li>
+            </ul></td>
+            <td><ul>
+              <li><code>recurring/</code></li>
+              <li><code>recurring/30 Nov 2100</code></li>
+            </ul></td>
+        </tr>
+        <tr>
+            <td>Outstanding fees</td>
+            <td><code>f/</code></td>
+            <td>Refer to <a href="#monetary-fields">Monetary fields</a> for more details.</td>
+            <td><code>lvl/J1</code></td>
+        </tr>
+        <tr>
+            <td>Homework</td>
+            <td><code>hw/</code></td>
+            <td>Maximum of 50 characters</td>
+            <td><code>hw/Test 1</code></td>
+        </tr>
+    </tbody>
+</table>
+
+* Additional fields of a lesson aside from those stated in the table are used for recording cancelled dates of a lesson. More details can be found in [Editing a lesson](#editing-a-lesson--ledit). 
+
+##### Monetary fields
+For all monetary fields (lesson rates and outstanding fees), we follow the [Singapore convention](https://www.dfa.cornell.edu/treasurer/cash-management/processinginternational/intl-currency) of using a decimal point '.' to separate dollars and cents, The values of these fields should only contain numbers, with at most one decimal point, and two decimal places and should not start or end with a decimal point.
+<br/>
 
 --------------------------------------------------------------------------------------------------------------------
 
 #### Adding a lesson: `ladd`
 
 Adds a lesson to the specified student in TAB, provided you do not have any other lessons scheduled at that time.
 
-Format: `ladd INDEX [recurring/[END_DATE]] date/dd MMM yyyy time/HHmm-HHmm subject/SUBJECT rates/LESSON_RATES [hw/HOMEWORK]…​`
+Format: `ladd INDEX [recurring/END_DATE] date/dd MMM yyyy time/HHmm-HHmm subject/SUBJECT rates/LESSON_RATES [f/OUTSTANDING_FEES] [hw/HOMEWORK]…​`
```

Comment:
should we change dd MMM yyyy -> START_DATE
and HHmm-HHmm -> Time range?

Reply:
Okay